indico/legacy/webinterface/tpls/RoomBookingRoomForm.tpl

```diff
@@ -44,6 +44,9 @@
     <div id="notifications_enabledCH" class="tip">
         ${ _("Enable/disable reminders for this room") }
     </div>
+    <div id="booking_limit_daysCH" class="tip">
+        ${ _("Period limit of days the room can be booked in one time (1 year by default)") }
```

Comment:
"Maximum number of days a booking for this room may last"?

Also, 1 year is a setting so we shouldn't show a hardcoded default here. We could just omit it for now or set the global default as a `placeholder` on the field so it shows up when there's no value.